### DISCUSSION TITLE
翻訳したら良さそうなCPANモジュールについて

### DISCUSSION THREAD

**Discussion by kfly8** [perldoc-jp/translation#39] [General]
## 背景、取り組みの理由

- [アンケートの結果、最近のPerl事情を掴みたいニーズ](https://blog.perlassociation.org/entry/2022/12/23/180000)があるよう
- そういった理由で、https://github.com/jpa-perl/wg-perl-document/issues/7 RFCの翻訳の活動もしている
- ただ、RFCの内容によっては、翻訳者にPerl力が必要でハードルが高いところがあった
- 最近のPerl事情を掴みつつ、翻訳のネタとして、Perlのモジュールの翻訳を検討するのはどうかと考えた
- そして、ワーキンググループで取り組むなら、どのモジュールを翻訳するかテーマ設定し、定期的に進捗を追うことで、アウトプットが安定するのではないかと思った

## 何をするか

次のようなページを参考に、翻訳したら良さそうなCPANモジュールについて、コメントをする。

- [コアモジュール](https://perldoc.perl.org/modules)
- [metacpanのリーダーボード](https://metacpan.org/favorite/leaderboard)
- [Perl Hackers Hub](https://gihyo.jp/list/group/Perl-Hackers-Hub)
- [Awesome Perl](https://github.com/hachiojipm/awesome-perl)

## 推奨行動

- 翻訳をするなど、進捗を出したらコメントをする
- そのコメントにいいね、エライといったリアクションをする

## 注意

- このdiscussionは元々、次のような issue https://github.com/jpa-perl/wg-perl-document/issues/17 で話していました
- 議論を目的とするなら、issueではなく、 discussionsが向いているのではないかと思い、このdiscussionを試しに開始してみました
- 議論することは本来の目的ではなく、翻訳を促進することが本来の目的です
- 翻訳が促進されない状況だった場合、このdiscussionをクローズする可能性は大いにあります

**Comment by kfly8** [comment]
[Feature::Compat::Class](https://perldoc.jp/docs/modules/Feature-Compat-Class-0.05/Class.pod)の翻訳が追加されました🎉🎉🎉

@bayashi ++++

**Comment by kfly8** [comment]
[WANTED] 翻訳を求む issueが切られました:mag: ぜひ翻訳してみてほしいです！

- https://github.com/jpa-perl/wg-perl-document/issues/21
- https://github.com/jpa-perl/wg-perl-document/issues/20
- https://github.com/jpa-perl/wg-perl-document/issues/19

**Reply by kfly8** [reply]
https://github.com/jpa-perl/wg-perl-document/issues/20 builtin の翻訳しますー

**Reply by nkchan** [reply]
#21 やりますー

**Reply by xtetsuji** [reply]
#19 やります！

**Comment by kfly8** [comment]
- https://github.com/jpa-perl/wg-perl-document/issues/17

現時点で、翻訳issueを切っていないモジュールだと、
mojo, class feature関連, Path::Tiny, Moo, JSON::PP, Type::Tinyあたりが、話に挙がってた。

**Comment by kfly8** [comment]
[WANTED] 翻訳を求むissueが切られたので、ぜひ翻訳してみてほしいです！

- https://github.com/jpa-perl/wg-perl-document/issues/18

**Comment by kfly8** [comment]
perl 5.38 も出たし、翻訳した良さそうなCPANモジュールも状況アップデートしたいなー。

**Comment by kfly8** [comment]
7/20時点で、accepted/implementedなPPCsの翻訳issueを切りましたー。
https://github.com/perldoc-jp/module-pod-jp/issues/49
